packages/prisma/src/prisma/schema.prisma

```diff
@@ -153,44 +164,95 @@ model MediaFile {
   updatedAt DateTime @updatedAt
 }
 
-// ----- QUESTION POOL -----
+// #endregion
+
+// ----- TAGS AND SKILLS -----
+// #region
+model Tag {
+  id   Int    @id @default(autoincrement())
+  name String
+
+  order Int @default(0)
+
+  originalId String? @unique
+
+  questions Element[]
+
+  owner   User   @relation(fields: [ownerId], references: [id], onDelete: Cascade, onUpdate: Cascade)
+  ownerId String @db.Uuid
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  @@unique([ownerId, name])
+}
+
+// #endregion
 
-enum QuestionType {
+// ----- ELEMENTS -----
+// #region
+enum ElementOrderType {
+  SEQUENTIAL
+  SHUFFLED
+  LAST_RESPONSE
+}
+
+enum ElementType {
+  // TODO: remove old types, to be migrated to new ones
   SC
   MC
   KPRIM
   FREE_TEXT
   NUMERICAL
+
+  // new types
+  CONTENT
+  FLASHCARD
+  QUESTION_SC
+  QUESTION_MC
+  QUESTION_KPRIM
+  QUESTION_FREE_TEXT
+  QUESTION_NUMERICAL
```

Comment:
Same as earlier - why do need to keep the old SC / MC / ... types, if we introduce QUESTION_SC, etc. - wouldn't it make sense to directly migrate and remove the old types within the same step?